Index: doit/doc/conf.py
===================================================================
--- doit.orig/doc/conf.py	2024-01-13 20:22:43.019798153 +0100
+++ doit/doc/conf.py	2024-01-13 22:44:11.091954950 +0100
@@ -89,7 +89,7 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 
-html_theme = 'press' #'sphinx_rtd_theme'
+html_theme = 'sphinx_rtd_theme'
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
